Marlin/ultralcd.cpp

```diff
@@ -1083,6 +1092,51 @@ void lcd_cooldown() {
 
 /**
  *
+ * "Change Filament" submenu
+ *
+ */
+static void lcd_change_filament_menu() {
+  START_MENU();
+  MENU_ITEM(back, MSG_MAIN);
+  MENU_ITEM(function, MSG_REMOVE_FILAMENT , lcd_retract_filament);
+  MENU_ITEM(submenu, MSG_INSERT_FILAMENT , lcd_insert_filament_menu);
+  END_MENU();
+}
+
+static void lcd_insert_filament_menu() {
+  START_MENU();
+  MENU_ITEM(back, MSG_CHANGE_FILAMENT);
+  MENU_ITEM_EDIT_CALLBACK(bool, MSG_INSERT_FILAMENT_SLOW ,&filamentSlowInsert ,lcd_insert_filament_slow);
+  if(!filamentSlowInsert)
+    MENU_ITEM(function, MSG_INSERT_FILAMENT  , lcd_insert_filament);
+  END_MENU();
+}
+
+static void lcd_insert_filament_slow(){
+ while (filamentSlowInsert) {
```

Comment:
Yes i'm looking there. Thanks, i just have one thing to say.
I saw that you make it something like "eject/insert while click". I just want to kno if its the best way to make it. Because my first approach was for printer that have bowden, that have that tube long way from the extruder, this way its size should be pre-settle-up and so, once setted and the eject/insert opction is chosed the filament will "walk" tru it itself, with no need of the user to keep pressed, and it would be really fast. The reason why i made the slow insert is just because if the insert start the fast one, the "feeder" may not catch the filament and end up not working, so slow would be to be used ultil the feed catch the filament and once it is ok, the user switch to the fast  insertion.
I just said it because the while click approach, at least to me, seemes to be focused to the printer who have direct.
Or maybe shall we have obtions for both?
LIke in configuration have a subsection of manual_filament_change where the user says if its using bowden or direct?